aait/backend/group-2A/Application/Features/Comment/Handlers/Commands/DeleteCommentCommandHandler.cs

```diff
@@ -15,19 +18,29 @@ public DeleteCommentCommandHandler(IUnitOfWork unitOfWork, IMapper mapper)
         _mapper = mapper;
         _unitOfWork = unitOfWork;
     }
-    
-    public async Task<Unit> Handle(DeleteCommentCommand request, CancellationToken cancellationToken)
+
+    public async Task<BaseCommandResponse> Handle(DeleteCommentCommand request, CancellationToken cancellationToken)
     {
+        var response = new BaseCommandResponse();
         var comment = await _unitOfWork.commentRepository.Get(request.Id);
-        
+
         if (comment == null)
         {
-            throw new Exception();
+            response.Success = false;
+            response.Message = "Comment Deletion Faild";
+            response.Errors = new List<string> { "Comment Not found" };
+        }
+        else
+        {
+            await _unitOfWork.commentRepository.Delete(comment);
+            await _unitOfWork.Save();
```

Comment:
check weather the database is affected or not : 
if  (await _unitOfWork.Save() == 0){
response.success = false;
response.Message = "server error";
}